tests/Test/Pager/Pagination/PaginatorTest.php

```diff
@@ -2,22 +2,19 @@
 
 namespace Test\Pager\Pagination;
 
-use Knp\Component\Pager\Exception\PagerBadArgumentException;
 use Knp\Component\Pager\Paginator;
 use Test\Tool\BaseTestCase;
 
-final class PaginatorTest extends BaseTestCase
-{
+final class PaginatorTest extends BaseTestCase {
```

Comment:
Parentheses must be on a new line, as per [PSR-2](https://www.php-fig.org/psr/psr-2/)